Issue with nan.h on windows (build fails)

I was having an issue where during the node-gyp rebuild process on my windows machine:

> ..\node-expat.cc(1): fatal error C1083: Cannot open include file: 'nan.h': No such file..

After running `npm install nan --save` I was still getting the issue, so I went into `node-expat.cc` and changed the include to be a relative path:

```
#include "node_modules/nan/nan.h"
```

Now everything is working fine. 

It seems like this is more of an issue with `node-gyp` not updating the `include_dirs` in the build process on windows machines properly. Just wanted to make anyone who runs into this aware.

--- a/binding.gyp
+++ b/binding.gyp
@@ -4,7 +4,7 @@
       'target_name': 'node_expat',
       'sources': [ 'node-expat.cc' ],
       'include_dirs': [
-        '<!(node -e \'require("nan")\')'
+        '<!(node -e "require(\'nan\')")'
       ],
       'dependencies': [
         'deps/libexpat/libexpat.gyp:expat'

